app/code/Magento/Theme/Model/View/Design.php

```diff
@@ -105,6 +105,7 @@ public function __construct(
      *
      * @param string $area
      * @return $this
+     * @deprecated
      */
     public function setArea($area)
     {
```

Comment:
@sivaschenko @sidolov @gabrieldagama  should we safe for remove this method ? Method defined in interface already removed but i just think we should keep method here for backward compatible

Reply:
In my we should clear legacy code in major release